backend/src/main/kotlin/rocks/didit/sefilm/Extensions.kt

```diff
@@ -57,3 +57,19 @@ internal fun TmdbMovieDetails.toMovie() =
 internal fun User.toLimitedUserInfo(): LimitedUserInfo {
   return LimitedUserInfo(this.id, this.name, this.firstName, this.lastName, this.nick, this.phone?.number, this.avatar)
 }
+
+internal fun Participant.toLimitedParticipant(): Participant {
+  val userId = currentLoggedInUser()
+  return when(this) {
+    is ForetagsbiljettParticipant -> if(this.userID == userId) this else LimitedParticipant(this.userID)
+    is SwishParticipant -> this
+    is LimitedParticipant -> this
+    else -> {
+      throw UnexpectedException("Oklar instans av participant");
```

Comment:
Use `RuntimeException` instead. Also I think error messages from the backend is in english. Maybe something we should change though...